src/library/basetracktablemodel.cpp

```diff
@@ -587,18 +594,23 @@ QVariant BaseTrackTableModel::rawSiblingValue(
         const QModelIndex& index,
         ColumnCache::Column siblingField) const {
     VERIFY_OR_DEBUG_ASSERT(index.isValid()) {
-        return QVariant();
+        return {};
     }
     VERIFY_OR_DEBUG_ASSERT(siblingField != ColumnCache::COLUMN_LIBRARYTABLE_INVALID) {
-        return QVariant();
+        return {};
     }
     const int siblingColumn = fieldIndex(siblingField);
-    if (siblingColumn < 0) {
+    VERIFY_OR_DEBUG_ASSERT(siblingColumn >= 0) {
         // Unsupported or unknown column/field
-        // FIXME: This should never happen but it does. But why??
-        return QVariant();
+        qWarning() << "Unknown sibling column"
+                   << siblingField
+                   << m_columnCache.columnName(siblingField)
+                   << "for model index"
+                   << index;
+        return {};
     }
     const QModelIndex siblingIndex = index.sibling(index.row(), siblingColumn);
+    DEBUG_ASSERT(siblingIndex.isValid());
```

Comment:
I did not change this code. It works as before. I only added more debug assertions to catch unexpected behavior during development.

The last debug assertion is just to verify that the resulting `siblingIndex` is valid, independent of what `rawValue()` does. `rawValue()` could be overridden and we cannot make any assumptions about its actual implementation.

Reply:
https://github.com/mixxxdj/mixxx/pull/13682/commits/a3ddee75c41c72bb736411d622b816add36746d5

I have reverted the unrelated changes, because other table models (e.g. history) still seem to be affected.